### DISCUSSION TITLE
Web rádio

### DISCUSSION THREAD

**Discussion by Vicente2024** [community/community#129713] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Bom dia tudo bem então eu estou aqui para apresentar a vocês usuários do Github minha web rádio bom ela não foi feita no Github mas eu estou procurando parceria com pessoas que tenha boas ideias e que goste de rádio eu sou desenvolvedor iniciante mas sei fazer muita coisa tenho muitas ideias muitas vezes só me falta um pouco de experiência bom aqui está a rádio HTTPS://primeiropost.com.br si alguém tiver algumas ideias e querer me ajudar ou patrocinar meu projeto estou a disposição para conversar obrigado

**Comment by Vicente2024** [comment]
Possíveis melhorias quero muito adicionar um bate papo feito na mão com banco de dados eu até fiz um mas era epen source e não ficou do jeito eu quero pretendo também adicionar uma rede social bom muita coisa a fazer espero que alguém possa me ajudar

**Comment by LiteBrite82** [comment]
Hi @Vicente2024 , thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
